packages/suite/src/hooks/wallet/coinmarket/form/common/useCoinmarketFiatValues.tsx

```diff
@@ -89,7 +84,7 @@ export const useCoinmarketFiatValues = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    if (!network || !accountBalance || !fiatCurrency) return null;
+    if (!accountBalance || !fiatCurrency) return null;
```

Comment:
using `networks[networkSymbol]` guarantees a result if TS is valid :slightly_smiling_face: